rest_framework/validators.py

```diff
@@ -138,7 +138,9 @@ def __call__(self, attrs):
         queryset = self.queryset
         queryset = self.filter_queryset(attrs, queryset)
         queryset = self.exclude_current_instance(attrs, queryset)
-        if queryset.exists():
+
+        # Ignore validation if any field is None
+        if None not in attrs.values() and queryset.exists():
```

Comment:
If a request comes in like

```json
{
  "unique_1": "something",
  "unique_2": "something else",
  "other": null
}
```

Wouldn't the uniqueness check be skipped because `None` is present in `attrs.values()`? `self.fields` contains the names of all of the fields that need to be checked, but `attrs` should contain the values of _all of the fields_ on the serializer that were passed in.